contracts/lib/ConsiderationConstants.sol

```diff
@@ -310,6 +310,8 @@ uint256 constant Conduit_execute_signature = (
     0x4ce34aa200000000000000000000000000000000000000000000000000000000
 );
 
+uint256 constant MaxUint120 = 0xffffffffffffffffffffffffffffff;
```

Comment:
This would be an option too:
```suggestion
uint256 constant MaxUint120 = uint256(type(uint120).max);
```

Reply:
Nice readability improvement!

Reply:
For the record, this doesn't actually work.

```
error[7615]: TypeError: Only direct number constants and references to such constants are supported by inline assembly.
```